src/IO/PlinkReader.cpp

```diff
@@ -75,17 +74,17 @@ void PlinkReader::getX(string filename, MatrixXf &X)
     long m = pio_num_samples( &plink_file );
     long n = pio_num_loci(&plink_file );
 
```

Comment:
should this always print?

Reply:
We should ask wenting about that one~ I will tell him

Reply:
The output only shows on the server, but you're right, it can be deleted.